DCF_Demo.md

```diff
@@ -131,35 +168,36 @@ For the next steps, you shall have the package key_vault.1 uploaded to the memor
 
 Query for existing interfaces on the device 
 ```
-az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "ipc.query.1.query" --mp "{}"
+az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "ipc.*.query.1:query" --mp "{}"
 
 // expected outcome
 {
   "payload": {
     "continuation_token": 655615,
     "result": [
-      "+ipc.1.query.1",
-      "+ipc.1.interface_manager.1",
-      "+dm.1.packages.1"
+      "*ipc.1.query.1",
+      "*ipc.1.interface_manager.1",
+      "*dm.1.packages.1"
     ]
   },
   "status": 200
 }
 
 ```
 
+**Note**: The `*` in front of the names represent that this is default package for that interface.
```

Comment:
>**Note**: The `*` in front of the names represent that this is default package for that interface [](http://example.com/codeflow?start=0&length=97)

nit: **Note**: A `*` in front of a name indicates that this is the default package for this interface.

Reply:
#Pending